images/cassandra/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM alpine:3.8
+
+ADD rook /usr/local/bin/
+
+ADD http://search.maven.org/remotecontent?filepath=org/jolokia/jolokia-jvm/1.6.0/jolokia-jvm-1.6.0-agent.jar /jolokia.jar
```

Comment:
does this jolokia package contain the Cassandra binaries also? if not, where does Cassandra get added to the container image?

Reply:
No it doesn't ! 
The operator uses the [official Cassandra image](https://hub.docker.com/_/cassandra/) from DockerHub. Of course, it gives the user the option to use his own repository.

This Dockerfile creates the image that is used for:
1. The Cassandra controller/operator
2. The sidecar that runs in the same pod as Cassandra, as a separate container.

It contains jolokia (and other plugins in the future), which the sidecar will copy to a shared volume location for the Cassandra container to use.